#! /bin/sh /usr/share/dpatch/dpatch-run
## 01makefile_sesman_fix.dpatch by  <bernat@neo.luffy.cx>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix Makefile for sesman installation

@DPATCH@

--- xrdp-0.4.0.orig/sesman/Makefile
+++ xrdp-0.4.0/sesman/Makefile
@@ -65,6 +66,7 @@
 	install sesman $(DESTDIRDEB)/usr/lib/xrdp/sesman
 	install startwm.sh $(DESTDIRDEB)/usr/lib/xrdp/startwm.sh
 	install sesman.ini $(DESTDIRDEB)/etc/xrdp/sesman.ini
+	make -C libscp installdeb DESTDIRDEB=$(DESTDIRDEB)
 
 os_calls.o: ../common/os_calls.c
 	$(CC) $(C_OS_FLAGS) ../common/os_calls.c
